scripts/docker-entrypoint.sh

```diff
@@ -1,19 +1,22 @@
 #!/bin/bash
-# Entrypoint script fro deploy production
+# Entrypoint script for deploy production
 # ---------------------------------------
 
 # Start XVfb
 if [[  -f /tmp/.X99-lock ]]; then
   rm /tmp/.X99-lock
 fi
```

Comment:
Isn't the docker container stateless? (why we are trying to remove that file from a previous run / reboot?)